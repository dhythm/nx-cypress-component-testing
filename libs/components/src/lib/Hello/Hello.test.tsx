import renderer from "react-test-renderer";
import { Hello } from "./";

describe("Hello", () => {
  it("Snap Shot", () => {
    const component = renderer.create(<Hello />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
