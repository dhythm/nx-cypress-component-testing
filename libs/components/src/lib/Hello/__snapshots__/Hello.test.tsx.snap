// Vitest Snapshot v1

exports[`Hello > Snap Shot 1`] = `
<div>
  Hello, World!
</div>
`;
